modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/AdvancedSettingsPage.kt

```diff
@@ -81,13 +75,13 @@ private fun SyncOnMeteredRow(
 ) {
     SettingRow(
         primaryText = stringResource(LR.string.settings_advanced_sync_on_metered),
-        secondaryText = stringResource(state.summary),
-        toggle = SettingRowToggle.Switch(state.isChecked),
+        toggle = SettingRowToggle.Switch(state.isChecked, state.isEnabled),
         modifier = modifier.toggleable(
             value = state.isChecked,
             role = Role.Switch
         ) { state.onCheckedChange(it) }
     )
+    SettingRow(primaryText = "", secondaryText = stringResource(LR.string.settings_advanced_sync_on_metered_summary))
```

Comment:
Once we shorten up the description text, I think it looks fine just setting this as the secondary text on the main `SettingRow` instead of creating this second one to keep the toggle switch at the top.